sonar-ws/src/main/java/org/sonarqube/ws/client/component/ComponentsService.java

```diff
@@ -65,7 +65,8 @@ public SearchWsResponse search(SearchWsRequest request) {
   public TreeWsResponse tree(TreeWsRequest request) {
     GetRequest get = new GetRequest(path(ACTION_TREE))
       .setParam(PARAM_COMPONENT_ID, request.getBaseComponentId())
-      .setParam(PARAM_COMPONENT, request.getBaseComponentKey())
+      .setParam("baseComponentKey", request.getBaseComponentKey())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getBaseComponentKey"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)